internal/datastore/mysql/datastore_test.go

```diff
@@ -90,14 +90,38 @@ func createDatastoreTest(b testdatastore.RunningEngineForTest, tf datastoreTestF
 }
 
 func TestMySQLDatastoreDSNWithoutParseTime(t *testing.T) {
+	t.Parallel()
 	_, err := NewMySQLDatastore("root:password@(localhost:1234)/mysql")
 	require.ErrorContains(t, err, "https://spicedb.dev/d/parse-time-mysql")
 }
 
-func TestMySQLDatastore(t *testing.T) {
+func TestMySQL5Datastore(t *testing.T) {
+	t.Parallel()
 	b := testdatastore.RunMySQLForTesting(t, "")
 	dst := datastoreTester{b: b, t: t}
 	test.All(t, test.DatastoreTesterFunc(dst.createDatastore))
+	additionalMySQLTests(t, b)
+}
+
+func TestMySQL8Datastore(t *testing.T) {
+	t.Parallel()
```

Comment:
I have concerns about parallel against running containers; are we sure this is stable?

Reply:
I'll remove it